code/src/pages/MovieDetails.js

```diff
@@ -0,0 +1,58 @@
+import React, { useEffect, useState } from 'react'
+import { useParams, Link } from 'react-router-dom'
+import { BackIcon } from '../components/BackIcon'
+import './moviedetails.css'
+
+export const MovieDetails = () => {
+  const { movieId } = useParams()
+  const [details, setDetails] = useState([])
+  const [loading, setLoading] = useState(false)
+
+  useEffect(() => {
+    setLoading(true)
+    fetch(`https://api.themoviedb.org/3/movie/${movieId}?api_key=01c5831a9741275652de4ceff9c6e22f&language=en-US`)
+      .then((res) => res.json())
+      .then((json) => {
+        console.log(json)
+        setDetails(json)
+        setLoading(false)
+      })
+  }, [movieId])
+
+  if (loading) {
+    return <h1>Loading...</h1>
```

Comment:
Hm maybe using something else than h1 here? I don't think you want "Loading..." to be the page title?